### PR TITLE
fix(bridge): Remove previous filter from URL

### LINKED ISSUE #7978: Bridge: The sequence screen does not always display the right query params
Steps to reproduce:
1. Select something in the filter
2. Refresh the page
3. Remove the filter and select something (an event, a sequence or a stage)
4. -->the old query param is here again

image

### PR DESCRIPTION
**Pull request by heinzburgstaller**
## This PR
- Fix component
- Add UI test

Fixes #7978

### CODE REVIEW

File: bridge/cypress/integration/sequences.spec.ts [hunk 1]
```diff
@@ -338,5 +338,22 @@ describe('Sequences', () => {
         .assertSequenceCount(1)
         .assertLoadOlderSequencesButtonExists(false);
     });
+
+    it.only('should not add the filter to the URL after it was cleared and the page was reloaded', () => {
```
Comment (Kirdock) [inline bridge/cypress/integration/sequences.spec.ts:342]: Please remove the `only` here

**Review by Kirdock — CHANGES_REQUESTED** [review]
We have two other places where the adjustment is needed.
1. https://github.com/keptn/keptn/blob/d82e52d220ddd39ea50764a6691c35084cde5430/bridge/client/app/_components/ktb-sequence-tasks-list/ktb-sequence-tasks-list.component.ts#L64
2. https://github.com/keptn/keptn/blob/d82e52d220ddd39ea50764a6691c35084cde5430/bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts#L427

File: bridge/cypress/integration/sequences.spec.ts (hunk 1 above)
Comment (heinzburgstaller) [reply bridge/cypress/integration/sequences.spec.ts]: Done

**Comment by heinzburgstaller** [comment]
> We have two other places where the adjustment is needed.
> 
>     1. https://github.com/keptn/keptn/blob/d82e52d220ddd39ea50764a6691c35084cde5430/bridge/client/app/_components/ktb-sequence-tasks-list/ktb-sequence-tasks-list.component.ts#L60-L69
> 
>     2. https://github.com/keptn/keptn/blob/d82e52d220ddd39ea50764a6691c35084cde5430/bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts#L423-L433

Done

**Pull request merged**